Public/Src/Cache/ContentStore/Distributed/Sessions/DistributedContentSession.cs

```diff
@@ -165,5 +180,51 @@ private async Task<PutResult> RegisterPutAsync(Context context, CancellationToke
 
             return putResult;
         }
+
+        private async Task<BoolResult> RequestProactiveCopyIfNeededAsync(OperationContext context, ContentHash hash)

```

Comment:
Need to wrap the entire thing into context.PerformOperartion.

Reply:
With its own counter for tracking the operation duration.

---
In reply to: [312225644](https://github.com/microsoft/BuildXL/pull/705#discussion_r312225644) [](ancestors = 312225644)